redwood-lazylayout-compose/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/LazyListState.kt

```diff
@@ -72,11 +67,18 @@ public class LazyListState {
     this.firstVisibleItemIndex = firstVisibleItemIndex
   }
 
-  public companion object {
+  @Composable
+  public fun rememberLazyListState(): LazyListState {
+    return rememberSaveable(saver = Saver) {
+      LazyListState()
+    }
+  }
```

Comment:
this feels a bit off for me :/

Reply:
You can keep the function top-level, but instead move the `private val Saver` to also be a top-level property. The function then will have access to it, and the `LazyListState` type won't need a companion at all.

Reply:
done!